packages/core/src/types.ts

```diff
@@ -1856,7 +1856,12 @@ export type ActorLogicFrom<T> = ReturnTypeOrValue<T> extends infer R
     : never
   : never;
 
-export type ActorRefFrom<T> = ReturnTypeOrValue<T> extends infer R
```

Comment:
This was overly complex and causing issues - we don't need the convenience of unpacking the return type here